src/libs/ReportUtils.ts

```diff
@@ -8116,6 +8118,29 @@ function isMoneyRequestReportPendingDeletion(reportOrID: OnyxEntry<Report> | str
     return parentReportAction?.pendingAction === CONST.RED_BRICK_ROAD_PENDING_ACTION.DELETE;
 }
 
+function navigateToLinkedReportAction(ancestor: Ancestor, isInNarrowPaneModal: boolean, canUserPerformWrite: boolean | undefined, isOffline: boolean) {
+    const isVisibleAction = shouldReportActionBeVisible(ancestor.reportAction, ancestor.reportAction.reportActionID, canUserPerformWrite);
```

Comment:
NAB - I'd prefer defining this below the `if (isInNarrowPaneModal) {` condition since that condition doesn't use this variable - what do you think @eVoloshchak ? Do we have a style guide saying we should declare most vars at the top of the function?